apps/main/src/llamalend/queries/market-rates.ts

```diff
@@ -32,12 +32,15 @@ const convertRates = ({
 const [isGetter, useAPI] = [true, true] as const
 
 export const { useQuery: useMarketRates } = queryFactory({
-  queryKey: ({ chainId, poolId }: BorrowApyParams) => [...rootKeys.pool({ chainId, poolId }), 'market-rates'] as const,
-  queryFn: async ({ poolId }: BorrowApyQuery) => {
-    const market = getLlamaMarket(poolId)
-    return market instanceof LendMarketTemplate
-      ? convertRates(await market.stats.rates(isGetter, useAPI))
-      : convertRates({ borrowApr: (await market.stats.parameters()).rate })
+  queryKey: ({ chainId, marketId }: MarketRateParams) =>
```

Comment:
what if we call it `params: MarketRateParams` and pass it to rootKeys like `...rootKeys.market(params)`? Saves the hassle of naming things.